scripts/test

```diff
@@ -10,10 +10,10 @@ echo
 if [[ -z ${CI} || -n ${CHECK_COVERAGE} ]]; then
     echo " -- RUNNING UNIT TESTS (WITH COVERAGE) --"
     # Test suite with coverage enabled
-    coverage run -m unittest discover tests
+    coverage run -m pytest tests
```

Comment:
We should use [pytest-cov](https://pypi.org/project/pytest-cov/) to run coverage.